spec/acceptance/exec_powershell_spec.rb

```diff
@@ -322,6 +322,17 @@
     it_should_behave_like 'should fail', pexception, /We are writing an error/i
   end
 
+  describe 'should error if timeout is exceeded' do
+    ptimeoutexception = <<-MANIFEST
+      exec{'PowershellException':
+        command  => 'Write-Host "Going to sleep now..."; Start-Sleep 5',
+        timeout  => 2,
+        provider => powershell,
+      }
+    MANIFEST
+    it_should_behave_like 'should fail', ptimeoutexception
```

Comment:
It doesn't seem like there's anyway to catch an error code here to validate that it failed due to a timeout, so is it worth matching against something like `timeout.*exceeded` to ensure this failed due to timeout? I suppose we have been going away from string matching...